package.json

```diff
@@ -6,18 +6,18 @@
 		"name": "Alex Kirk",
 		"email": "alex.kirk@automattic.com"
 	},
-	"keywords": [
-	],
+	"keywords": [],
 	"dependencies": {
-		"jed": "*",
 		"JSON2": "*",
-		"debug": "1.0.1"
+		"debug": "1.0.1",
+		"jed": "*"
 	},
 	"devDependencies": {
 		"better-assert": "*",
 		"browserify": "*",
 		"chai": "^2.1.2",
-		"jsdom": "^2.0.0",
+		"jquery": "^3.3.1",
```

Comment:
This made me realize - in calypso we load community-translator with [loadjQueryDependentScript](https://github.com/Automattic/wp-calypso/blob/5db7b0348ed81466253ba04ef9ac9b158497ffe7/client/lib/translator-jumpstart/index.js#L244) - but we don't really need it since it's bundled in, right?

Reply:
Actually, nope. It fails without it. `ReferenceError: jQuery is not defined`, and I see now that's it's only in the dev dependencies, so nevermind :)

Reply:
Thanks for testing! Yeah, it's just there to simulate the browser environment for tests. :)